recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
-{% set version = "20160523b" %}
+{% set version = "20170520" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
   fn: texlive-{{ version }}-source.tar.xz
-  url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: a8b32ca47f0a403661a09e202f4567a995beb718c18d8f81ca6d76daa1da21ed
+  url: http://mirrors.concertpass.com/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
```

Comment:
I'd revert this change, mirrors.ctan.org is automatically redirected to one of the many mirrors around the world according to the user position and is the link used at https://www.ctan.org/tex-archive/systems/texlive/Source